Spectrogram app example is broken

First simple issue: SciPy renamed the `simps` function to `simpson`.  This diff fixes that problem
```diff
diff --git a/examples/server/app/spectrogram/audio.py b/examples/server/app/spectrogram/audio.py
index 8926f5eed1..b9f1ff2d5d 100644
--- a/examples/server/app/spectrogram/audio.py
+++ b/examples/server/app/spectrogram/audio.py
@@ -2,7 +2,7 @@ from time import sleep

 import numpy as np
 from scipy.fft import fft
-from scipy.integrate import simps
+from scipy.integrate import simpson

 NUM_SAMPLES = 1024
 SAMPLING_RATE = 44100.
@@ -32,7 +32,7 @@ try:
                 signal = raw_data / 32768.0
                 spectrum = abs(fft(signal))[:int(NUM_SAMPLES/2)]
                 power = spectrum**2
-                bins = simps(np.split(power, NUM_BINS))
+                bins = simpson(np.split(power, NUM_BINS))
                 data['values'] = signal, spectrum, bins
             except Exception:
                 continue
```

However, the extension is broken due to BokehJS changes as well:
```
025-02-08 16:59:42,909 Starting Bokeh server version 3.7.0.dev5+28.g65e644fd (running on Tornado 6.4.2)
2025-02-08 16:59:42,910 User authentication hooks NOT provided (default user enabled)
2025-02-08 16:59:42,912 Bokeh app running at: http://localhost:5006/spectrogram
2025-02-08 16:59:42,912 Starting Bokeh server with process id: 19715
Exception in thread Thread-1 (update_audio_data):
Traceback (most recent call last):
  File "/Users/bryan/anaconda3/envs/dev312/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Users/bryan/anaconda3/envs/dev312/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/bryan/work/bokeh/examples/server/app/spectrogram/audio.py", line 67, in update_audio_data
    bins = simps(np.split(power, NUM_BINS))
           ^^^^^
NameError: name 'simps' is not defined. Did you mean: 'simpson'?
Compilation failed:

spectrogram/waterfall.ts:8:14 - error TS2515: Non-abstract class 'WaterfallRendererView' does not implement inherited abstract member _paint from class 'RendererView'.

8 export class WaterfallRendererView extends RendererView {
               ~~~~~~~~~~~~~~~~~~~~~
spectrogram/waterfall.ts:48:42 - error TS2339: Property 'request_render' does not exist on type 'WaterfallRendererView'.

48     this.connect(this.model.change, this.request_render)
```
@mattpap please advise if you can.

--- a/examples/server/app/spectrogram/audio.py
+++ b/examples/server/app/spectrogram/audio.py
@@ -2,7 +2,7 @@ from time import sleep
 
 import numpy as np
 from scipy.fft import fft
-from scipy.integrate import simps
+from scipy.integrate import simpson
 
 NUM_SAMPLES = 1024
 SAMPLING_RATE = 44100.
@@ -32,7 +32,7 @@ try:
                 signal = raw_data / 32768.0
                 spectrum = abs(fft(signal))[:int(NUM_SAMPLES/2)]
                 power = spectrum**2
-                bins = simps(np.split(power, NUM_BINS))
+                bins = simpson(np.split(power, NUM_BINS))
                 data['values'] = signal, spectrum, bins
             except Exception:
                 continue
@@ -64,6 +64,6 @@ except ImportError:
 
             spectrum = abs(fft(signal))[:int(NUM_SAMPLES/2)]
             power = spectrum**2
-            bins = simps(np.split(power, NUM_BINS))
+            bins = simpson(np.split(power, NUM_BINS))
             data['values'] = signal, spectrum, bins
             sleep(1.0/12)

--- a/examples/server/app/spectrogram/waterfall.ts
+++ b/examples/server/app/spectrogram/waterfall.ts
@@ -45,10 +45,10 @@ export class WaterfallRendererView extends RendererView {
 
   connect_signals(): void {
     super.connect_signals()
-    this.connect(this.model.change, this.request_render)
+    this.connect(this.model.change, this.request_paint)
   }
 
-  protected _render(): void {
+  protected _paint(): void {
     const {ctx} = this.layer
     ctx.save()
     ctx.imageSmoothingEnabled = false

--- a/tests/examples.yaml
+++ b/tests/examples.yaml
@@ -25,6 +25,7 @@
   skip: [
     "faces",
     "simple_hdf5",
+    "spectrogram", # Custom models not working in CI test harness
     "stocks",      # Bokeh.index not available, can't compute bbox
     "surface3d",   # Bokeh.index not available, can't compute bbox
   ]

--- a/tests/examples.yaml
+++ b/tests/examples.yaml
@@ -25,9 +25,8 @@
   skip: [
     "faces",
     "simple_hdf5",
-    "spectrogram",
-    "stocks",      # Bokeh.index not avaiable, can't compute bbox
-    "surface3d",   # Bokeh.index not avaiable, can't compute bbox
+    "stocks",      # Bokeh.index not available, can't compute bbox
+    "surface3d",   # Bokeh.index not available, can't compute bbox
   ]
 
 - path: "examples/styling/**"

